src/lib/cache/deep-research-cache.ts

```diff
@@ -0,0 +1,65 @@
+import IORedis from "ioredis";
+import { DeepResearchResult } from "../schemas/deep-research";
+import { env } from "~/utils/env";
+
+// Redis client setup with default configuration
+const redisClient = new IORedis(env.REDIS_URL, {
```

Comment:
Don't we have a redis client defined elsewhere already? We should not redefine these things.

Will this conflict with BullMQ's Redis connection?